server/db/actions/Log.js

```diff
@@ -0,0 +1,72 @@
+import dbConnect from "../dbConnect";
+import Log from "../models/Log";
+import Dog from "../models/Dog";
+
+export async function createLog(logData) {
+  await dbConnect();
```

Comment:
I would wrap this in a try/catch with a custom error message that is more user friendly. Right now, if this function errors because we are unable to connect to the db, the mongoose error message is thrown (and we don't want the client seeing these dev-specific errors). 

Instead, we can do something like this:
```js
try {
    await dbConnect();
} catch (e) {
    throw new Error("Unable to create dog at this time, please try again");
}
```